apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx

```diff
@@ -187,7 +203,16 @@ export class WorkshopEnrollmentSchoolInfo extends React.Component {
           <td>{i + 1}</td>
           <td>{enrollment.first_name}</td>
           <td>{enrollment.last_name}</td>
-          <td>{enrollment.email}</td>
+          <td>
+            {enrollment.email}
+            {application_url && (
+              <p>
+                <a href={application_url} target="_blank">
+                  Application
```

Comment:
I wonder if this should say "See Application" or something with a clearer verb.  Also might bounce this design off of @breville - he has good design instincts and might have a better idea for where to put this link in the table.

Reply:
I'm open to design suggestion @breville  :)

Reply:
If we look at the preceding screen, we have precedent for action buttons with a verb in them:

![Screenshot 2020-03-13 11 18 50](https://user-images.githubusercontent.com/2205926/76648752-80a4e000-651c-11ea-94b4-9e4d540a0737.png)

So at minimum we should say "View application", and I think we should use this kind of compact button too.

To be truly consistent we might want to consider having a new column, rather than reusing an existing one, but in this case it seems reasonable to reuse the existing column so that we don't squash all the others.

Reply:
That's the main reason I want to reuse a column. The enrollment table is already quite tight, adding one more column will force user to scroll horizontally.  I'll change the text to **"View Application"**.